src/content/docs/fr/guides/cms/keystatic.mdx

```diff
@@ -204,7 +206,7 @@ const posts = await getCollection('posts')
 
 ### Affichage d'un seul article
 
-Pour afficher le contenu d'un article individuel, vous pouvez importer et utiliser le composant `<Content />` pour [rendre votre contenu en HTML](/fr/guides/content-collections/#rendre-le-contenu-en-html) :
+Pour afficher le contenu d'un article individuel, vous pouvez importer et utiliser le composant `<Content />` pour [rendre votre contenu en HTML](/fr/guides/content-collections/#rendre-le-contenu-du-corps) :
```

Comment:
We should keep the previous anchor in the meantime:
```suggestion
Pour afficher le contenu d'un article individuel, vous pouvez importer et utiliser le composant `<Content />` pour [rendre votre contenu en HTML](/fr/guides/content-collections/#rendre-le-contenu-en-html) :
```
(I updated the section in #10145 and the anchor will be `restitution-du-contenu` but with the broken links... I guess this PR will be merged later)